third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-clear-data-warning.js

```diff
@@ -8,6 +8,10 @@
 
   await TestRunner.loadTestModule('lighthouse_test_runner');
   await TestRunner.showPanel('lighthouse');
+  await TestRunner.RuntimeAgent.invoke_evaluate({
+    expression: 'webSqlPromise',
+    awaitPromise: true,
+  });
```

Comment:
I tried fixing this too. I did this:

![image](https://user-images.githubusercontent.com/4071474/150215092-9210759e-bd4e-462b-b81a-493bddee39ff.png)

I was trying to make LH wait for the DB by waiting for first paint.

But... this went from flaky to never seeing the DB usage. I never figured out why.

Reply:
_ooooh_, I see what you've done. You made the DB get initiated before LH even runs.

So how was this ever working before? :)

Reply:
> So how was this ever working before?

It was flaky, sometimes the row would be added to the table in time, and sometimes it wouldn't. In CI it usually got added in time so I just kinda ignored the problem when it showed up locally.

Reply:
> To make that more clear can you move this to right after the navigate?

Move what to the right? Sorry don't understand.

Reply:
Not "to the right". "right after" `TestRunner.navigatePromise`.

Reply:
🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃 🙃